frontend/src/components/ExternalDb.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <section class="section extended-section">
+    <div v-if="externalDb" class="container is-fullhd">
+      <h3 class="title">
+        {{ externalDb.dbName }} {{ components[0].componentType }} {{ externalDb.externalId }}
+      </h3>
+      <h4 class="subtitle">
+        The following Metabolic Atlas
+        {{ components.length === 1 ? 'component is' : 'components are' }}
+        associated with this external DB reference.
+      </h4>
```

Comment:
Matching end tag for the paragraph, and some extra space that is now gone after removing the `subtitle` class.
```suggestion
      </p>
      <br>
```